packages/fixtures/src/fields.spec.ts

```diff
@@ -15,15 +15,15 @@ describe('Defaults', () => {
     const overrideToken = 'a-very-different-token';
     const originalToken = fields.token;
 
-    expect(events.blockButtonAction()).toEqual(
+    expect(actions.blockButtonAction()).toEqual(
       expect.objectContaining({
         token: originalToken,
       }),
     );
 
     fields.token = overrideToken;
 
-    expect(events.blockButtonAction()).toEqual(
+    expect(actions.blockButtonAction()).toEqual(
```

Comment:
is `blockButtonAction` a good name now that it's a method on `actions`?

Reply:
Probably not